IdentityCore/src/network/MSIDHttpRequest.m

```diff
@@ -80,6 +85,18 @@ - (void)sendWithBlock:(MSIDHttpRequestDidCompleteBlock)completionBlock
                                                       data:data
                                                      error:error];
           if (error)
+          {
+              if (completionBlock) { completionBlock(nil, error); }
+          }
+          else if (httpResponse.statusCode == 200)
+          {
+              id responseObject = [self.responseSerializer responseObjectForResponse:httpResponse data:data error:&error];
+              
+              MSID_LOG_VERBOSE(self.context, @"Parsed response: %@, error %@", _PII_NULLIFY(responseObject), _PII_NULLIFY(error));
```

Comment:
nit: please still print error domain and error code. PII nullify will just print if the object is nil or not.

Reply:
done